chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/SearchRequest.java

```diff
@@ -144,4 +147,62 @@ public Boolean getHasHadSurveillance() {
 	public void setHasHadSurveillance(Boolean hasHadSurveillance) {
 		this.hasHadSurveillance = hasHadSurveillance;
 	}
+	
+	@Override
+	public int hashCode() {
+		final int prime = 31;
+		int result = 1;
+		result = prime * result
+				+ ((searchTerm == null) ? 0 : searchTerm.hashCode());
+		result = prime * result

```

Comment:
Do you mean to do "=" here and reassign the result of "result" or did you mean to do "+=" to add everything together? Overriding equals and hashcode can affect tons of underlying behavior so it *has* to be right.